src/it/scala/com/exasol/cloudetl/kinesis/KinesisAbstractIntegrationTest.scala

```diff
@@ -98,7 +98,6 @@ trait KinesisAbstractIntegrationTest extends AnyFunSuite with BeforeAndAfterAll
     statement.execute(
       s"""CREATE OR REPLACE JAVA SET SCRIPT KINESIS_METADATA (...)
          |EMITS (KINESIS_SHARD_ID VARCHAR(130), SHARD_SEQUENCE_NUMBER VARCHAR(2000)) AS
-         |     %jvmoption -Dcom.amazonaws.sdk.disableCbor=true;
```

Comment:
I thought this should not be removed for integration tests?

Reply:
This one surprisingly works without this option. The data extraction doesn't.